profile-widget/profilewidget2.cpp

```diff
@@ -480,7 +469,6 @@ void ProfileWidget2::setPlanState(const dive *d, int dc)
 	setProfileState(d, dc);
 	mouseFollowerHorizontal->setVisible(true);
 	mouseFollowerVertical->setVisible(true);
-	disconnectTemporaryConnections();
```

Comment:
Have you checked if this may lead to double connection? Those can be very painful.

Reply:
Yes - `setProfileState()` right above already calls `disconnectTemporaryConnections()`.

Reply:
Ah, ok.

This obviously collides with my efforts to port the profile to QtQuick, but oh weill...

Reply:
Apologies for the collision. I am on a quest to make the addition of gases and gas switches work for manually added dives, and in the process I have been cleaning up some less than ideal implementation details I've come across.